nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -332,7 +365,7 @@ public DocumentRef getTopLevelCommentAncestor(CoreSession s, DocumentRef documen
     /**
      * Gets or creates the 'Comments' folder, this folder will be under the document being commented and it contains the
      * whole comments of the first document that we comment.
-     * 
+     *
```

Comment:
I can't separate this in a single commit. Git refuses to commit space only diffs...so unless I turn down white space trimming (and so does everybody editing the file without cleanup commit) this is going to stay...